neurons/validator/validator.py

```diff
@@ -257,21 +281,60 @@ def __init__(self):
         self.background_timer = BackgroundTimer(60, background_loop, [self, True])
         self.background_timer.daemon = True
         self.background_timer.start()
-        
+
         # Create a Dict for storing miner query history
         try:
-            self.miner_query_history_duration = {self.metagraph.axons[uid].hotkey:float('inf') for uid in range(self.metagraph.n.item())}
+            self.miner_query_history_duration = {
+                self.metagraph.axons[uid].hotkey: float("inf")
+                for uid in range(self.metagraph.n.item())
+            }
         except:
             pass
         try:
-            self.miner_query_history_count = {self.metagraph.axons[uid].hotkey:0 for uid in range(self.metagraph.n.item())}
+            self.miner_query_history_count = {
+                self.metagraph.axons[uid].hotkey: 0
+                for uid in range(self.metagraph.n.item())
+            }
         except:
             pass
         try:
-            self.miner_query_history_fail_count = {self.metagraph.axons[uid].hotkey:0 for uid in range(self.metagraph.n.item())}
+            self.miner_query_history_fail_count = {
+                self.metagraph.axons[uid].hotkey: 0
+                for uid in range(self.metagraph.n.item())
+            }
         except:
             pass
 
+        # Load Moving Average and Weight Dataframe
```

Comment:
Is this just for testing?

Reply:
It was also for logging purposes for each validator but now that we have the moving_averages live chart we can remove this.